SecretSanta/test/SecretSanta.Api.Tests/Controllers/BaseApiControllerTests.cs

```diff
@@ -61,7 +61,7 @@ public async Task Get_WhenEntityExists_ReturnsItem()
             service.Items.Add(entity);
             BaseApiController<TDto, TInputDto> controller = CreateController(service);
 
-            IActionResult result = await controller.Get(entity.Id);
+            var result = await controller.Get(entity.Id);
 
             var okResult = result as OkObjectResult;
```

Comment:
```suggestion
            var okResult = result.Result as OkObjectResult;
```
This should fix the failing tests.